src/components/victory-bar.jsx

```diff
@@ -265,24 +281,29 @@ class VBar extends React.Component {
   }
 
   _getAttributes(props, index) {
-    const attributes = props.dataAttributes && props.dataAttributes[index] ?
+    let attributes = props.dataAttributes && props.dataAttributes[index] ?
       props.dataAttributes[index] : props.dataAttributes;
+    if (attributes) {
+      attributes.fill = attributes.fill || this.getColor(props, index);
+    } else {
+      attributes = {fill: this.getColor(props, index)};
+    }
     const requiredAttributes = {
       name: attributes && attributes.name ? attributes.name : "data-" + index
     };
     return _.merge(requiredAttributes, attributes);
   }
 
-  containsStrings(collection) {
-    return _.some(collection, function (item) {
-      return _.isString(item);
-    });
+  getColor(props, index) {
+    const useColorScale = Array.isArray(props.colorScale) ?
```

Comment:
"useColorScale" feels a bit clunky/not quite accurate to me; open to suggestions